largestProduct.js

```diff
@@ -1,6 +1,40 @@
 class LargestProduct {
   constructor(number) {
+    this.getLargestProduct = function() {
+      let product = 0;
+      let arr = number
+        .toString(10)
+        .split('')
+        .map(Number);
 
+      for (let i = 0; i < arr.length; i++) {
```

Comment:
You probably want to limit this to `arr.length - 5` since once it gets to about arr.length - 4 there's nothing in the arr[i + 4] index of the array. So you're basically doing, someNumber * someNumber * someNumber * someNumber * undefined.